mod/01_rating/set/type/metric_value.rb

```diff
@@ -144,35 +157,34 @@ def legend args
   end
 
   view :timeline_data do |args|
-    year  =  content_tag(:span, card.cardname.right, :class=>'metric-year')
-    value_card = card.fetch(:trait=>:value)
-    value =  _render_modal_details(args) 
-    value << content_tag(:span, legend(args), :class=>'metric-unit')
-
-    line   =  content_tag(:div, '', :class=>'timeline-dot')
-    line   << content_tag(:div, '', :class=>'timeline-line') if args[:connect]
+    year  =  content_tag(:span, card.cardname.right, class: 'metric-year')
+    # value_card = card.fetch(trait: :value)
+    value =  _render_modal_details(args)
+    value << content_tag(:span, legend(args), class: 'metric-unit')
 
-    credit = wrap_with :div, :class=>'td credit' do
-            [
-              nest(card, :view=>:core, :structure=>'creator credit'),
-              _optional_render(:source_link, args, :hide)
-            ]
-          end
+    line   =  content_tag(:div, '', class: 'timeline-dot')
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.